### PR TITLE
Add a certificate generation app

### LINKED ISSUE #474: Add Certificate Generation to Yaksh

### PR DESCRIPTION
**Pull request by ankitjavalkar**
Closes #474 

Pending: Needs a verification system, will be resolved as a part of #475

### CODE REVIEW

File: certificates/models.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    def create_certificate_html(self, student):
+        course_status = CourseStatus.objects.get(user=student, course=self.course)
+        context = {'student': student, 'course': self.course, 'course_status': course_status}
+        filepath = get_cert_template_dir(self)
+        template = os.path.join(filepath, CERTIFICATE_FILE_NAME)
+        return render_certificate_template(template, context)
+
+    def create_certificate_pdf(self, student):
+        html = self.create_certificate_html(student, self.course)
```
Comment (maheshgudi) [inline certificates/models.py]: create_certificate_html takes 1 arg,  that's student!

Comment (ankitjavalkar) [reply certificates/models.py]: Fixed

File: certificates/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
[the remark this answers is not in the document]
Comment (ankitjavalkar) [reply certificates/admin.py]: Fixed

**Comment by ankitjavalkar** [comment]
Closing for now. Will update and resend PR.

**Pull request closed without merging**